base/macros.hpp

```diff
@@ -123,7 +123,7 @@ inline void noreturn() { std::exit(0); }
 #if PRINCIPIA_COMPILER_CLANG    ||  \
     PRINCIPIA_COMPILER_CLANG_CL ||  \
     PRINCIPIA_COMPILER_GCC
-#  define FORCE_INLINE [[gnu::always_inline]]  // NOLINT(whitespace/braces)
+#  define FORCE_INLINE [[gnu::always_inline]] inline  // NOLINT(whitespace/braces)
```

Comment:
Lint says: 'Lines should be <= 80 characters long`.

Just remove `(whitespace/braces)` (keeping only `NOLINT`) and it should fit.